auto-cpufreq.conf-example

```diff
@@ -7,6 +7,9 @@ governor = performance
 # EPP: see available preferences by running: cat /sys/devices/system/cpu/cpu0/cpufreq/energy_performance_available_preferences
 energy_performance_preference = performance
 
+# PP: see available preferences by running: cat /sys/firmware/acpi/platform_profile_choices
```

Comment:
Let's change this part:

> PP: see available preferences by running: cat /sys/firmware/acpi/platform_profile_choices

so it looks like something like this wherever it's mentioned:

> Platform Profiles 
https://www.kernel.org/doc/html/latest/userspa-ce-api/sysfs-platform_profile.html

> See available options by running:
> cat /sys/firmware/acpi/platform_profile_choices